src/api/endpoints/logged_activities/secretary_review.py

```diff
@@ -41,7 +44,15 @@ def put(self, logged_activity_id):
         if not (payload.get('status') in ['pending', 'rejected']):
             return response_builder(dict(message='Invalid status value.'),
                                     400)
+
+        roles = User.query.filter(User.roles.any(Role.name=="success ops")).all()
+        users = User.query.all()
+        message = "New society points approved by society secretary. Go to https://societies.andela.com/u/verify-activities to review points" # noqa: E501
```

Comment:
If we have access to the Society whose secretary has approved points (as well as the points), this message could become :
`The society secretary for iStelle has approved an activity worth 200 points. Visit https://societies.andela.com/u/verify-activities to approve or reject these points.`

This provides more value at a glance. This should be possible. Do advise @jimmykamau .

Reply:
This is possible, it will be implemented.